jest/integration.config.js

```diff
@@ -0,0 +1,34 @@
+module.exports = {
+  rootDir: '../',
+  collectCoverageFrom: ['packages/**/*.js', '!src/**/*.test.js', '!src/**/*.spec.js', '!src/**/*.e2e.js'],
+  cacheDirectory: '<rootDir>/jest_cache',
+  // testRegex: '__tests__/.*\\.test\\.js$',
+
+  moduleFileExtensions: ['ts', 'js', 'node', 'json'],
+  transform: {
+    '^.+\\.(js|jsx)$': 'babel-jest',
+    '^.+\\.(ts|tsx)$': 'ts-jest',
+  },
+  testMatch: ['/**/?(*.)+(test).[jt]s?(x)'],
+  // testEnvironment: 'node',
+  // testEnvironment: './jest/env.js',
+  moduleNameMapper: {
+    '@cennznet/api(.*)$': '<rootDir>/packages/api/src/$1',
+    '@cennznet/crml-attestation(.*)$': '<rootDir>/packages/crml-attestation/src/$1',
```

Comment:
`crml-*` packages could be removed from the initial jest config since they aren't required for #70 or maybe the actual packages should be removed in #70

Reply:
`moduleNameMapper ` are copied from the current legacy settings.

Could comment them for now, and remove `crml-*` together with testing stuff in a separated PR, as current PR is already so big